micronaut-camunda-bpm-feature/src/main/java/info/novatec/micronaut/camunda/bpm/feature/MnProcessEngineConfiguration.java

```diff
@@ -76,6 +78,14 @@ protected void initTransactionContextFactory() {
         }
     }
 
+    @Override
+    protected void initJobExecutor(){
+        JobExecutor jobExecutor = new DefaultJobExecutor();
+        jobExecutor.setWaitTimeInMillis(300);
```

Comment:
remove setWaitTimeInMillis - this was only hacking code!

please call jobExecutorCustomizer.customize(jobExecutor);

here instead. You will need to add the JobExecutorCustomizer to the constructor and set a new field: jobExecutorCustomizer

Reply:
Done